db/index.js

```diff
@@ -33,6 +33,37 @@ let totalReviewsSchema = new mongoose.Schema({
 
 });
 
-module.exports = { db, reviewsSchema, totalReviewsSchema };
+const Reviews = mongoose.model('Reviews', reviewsSchema);
+const TotalReviews = mongoose.model('TotalReviews', totalReviewsSchema);
+
+const getUserReview = function(id) {
+  return new Promise((resolve, reject) => {
+    Reviews.findOne({courseNumber: id}, (err, results) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(results);
+      }
+    });
+  });
+};
+
+const getTotalReviewScore = function (id) {
+  return new Promise((resolve, reject) => {
+    TotalReviews.findOne({ courseNumber: id }, (err, results) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(results);
+      }
+    });
+  });
+};
+
+module.exports = {
+  db: db,
```

Comment:
db is defined on line 6. I see what you mean, about object destructuring and how the redundancy of same key/values isn't necessary. I believe I had it that way originally, but changed it because I was having some errors, that probably were related to something else. I agree that it looks cleaner to have it destructured. Gonna try to go back to that and see if it causes any problems.